lib/ruby_lsp/test_reporters/minitest_reporter.rb

```diff
@@ -116,6 +123,45 @@ def report
     def handle_spec_test_id(method_name, line)
       method_name.gsub(/(?<=test_)\d{4}(?=_)/, format("%04d", line.to_s))
     end
+
+    private
+
+    #: (untyped, String) -> [URI::Generic?, Integer?, String?]
+    def extract_test_location(test_context, method_name)
+      if test_context.respond_to?(:instance_method)
```

Comment:
Every class responds to `instance_method`. Is there a more specific check we could use here?